test/explain/test_graph_generator.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import torch
+
+from torch_geometric.datasets import ExplainerDataset
+from torch_geometric.datasets.generators import Motif
+from torch_geometric.datasets.generators.graph_generator import GraphGenerator
+from torch_geometric.utils import barabasi_albert_graph
+
+
+class BAGraph(GraphGenerator):
```

Comment:
Should we define the `BAGraph` here or should we move it to the generators.

Reply:
Good point. I assume it is better to remove this class and use the implemented one which is in the other branch. I'll update!